tests/logout.test.js

```diff
@@ -0,0 +1,35 @@
+import LoginPage from '../pages/login.page';
+import LogoutPage from '../pages/logout.page';
+import HomePage from '../pages/home.page';
+
+describe('Logout Test Cases', () => {
+
+  it('Should logout successful when click on the Logout button', async () => {
+    await LoginPage.open('login');
+    await LoginPage.completeFields(LoginPage.usernameInput, 'jsmith@demo.io');
+    await LoginPage.completeFields(LoginPage.passwordInput, 'Demo123!');
+    await LoginPage.logInWithSubmitButton();
+
+    await LoginPage.clickElement(LogoutPage.profileIcon);
+    await LoginPage.clickElement(LogoutPage.logoutButton);
+    expect(LogoutPage.successfulLogoutMessage).toHaveText('Logout completed.');
+  });
+  
+  it('Should close a session when you try to have 2 simultaneously', async () => {
+    await LoginPage.open('login');
+    await LoginPage.completeFields(LoginPage.usernameInput, 'jsmith@demo.io');
+    await LoginPage.completeFields(LoginPage.passwordInput, 'Demo123!');
+    await LoginPage.logInWithSubmitButton();
+
+    await browser.newWindow('login');
+    await LoginPage.completeFields(LoginPage.usernameInput, 'nsmith@demo.io');
+    await LoginPage.completeFields(LoginPage.passwordInput, 'Demo123!');
+    await LoginPage.logInWithSubmitButton();
```

Comment:
This pattern (filling in fields for login inputs and pressing a button or enter) repeats throughout most test, and, in particular, repeats in this test. This is a good sign that they maybe belong together in a higher-level operation that abstracts them out a bit more. Maybe `loginInWithSubmitButton()` and `logInWithEnter()` should include the completing fields operation, so you pretty much directly use them to fullfill the whole login process: Something like: `LoginPage.logInWithSubmitButton('jsmith@demo.io', 'Demo123!')` essentially parameterizing username and password, and then internally using `completeFields()` and clicking the specific button / pressing the specific key after that. This will remove that duplication, shorten and improve readability of the tests